CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ## [1.X.X] - 20XX-XX-XX
 
+### Changed
+- energy transformation (#1543)
```

Comment:
In Line 18 there is already a "changed" section. Please include your changes there below.